.eslintrc.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
     'no-console': 'error', // airbnb is using warn
     'no-param-reassign': 'off',
     'no-prototype-builtins': 'off',
+    'no-use-before-define': ['error', { 'functions': false }], // airbnb have functions: true, annoying
```

Comment:
Yeah, I see that side too -- but on the other hand it can be confusing to open a file and it's hard to find the `describe()` in a test because there's a bunch of helper functions first. In that case, you don't even know what they are for yet.